docs/core/compatibility/interop/9.0/cet-support.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Breaking change: CET supported by default"
+description: Learn about the breaking change in interop in .NET 8 where CET is supported by default.
```

Comment:
@gewarren I've just noticed that it mentions .NET 8 here. The change is in .NET 9 though.